test/unittest/unittest_nntrainer_tensor.cpp

```diff
@@ -4460,6 +4460,69 @@ TEST(nntrainer_Tensor, dequantize_05_n) {
   EXPECT_THROW({ input.dequantize(output, 1); }, std::invalid_argument);
 }
 
+TEST(nntrainer_neon_experimental, trigonometric_simd) {
```

Comment:
just an idea. think it would be better to split this into two separate tests (cos and sin tests).

Reply:
Sounds more reasonable!
Fixed.